vizro-core/src/vizro/models/_components/graph.py

```diff
@@ -40,6 +41,14 @@ class Graph(VizroBaseModel):
     figure: CapturedCallable = Field(
         ..., import_path="vizro.plotly.express", mode="graph", description="Function that returns a plotly `go.Figure`"
     )
+    title: str = Field("", description="Title of the `Graph`")
+    header: str = Field(
+        "",
+        description="Markdown text positioned below the component title, compliant with the CommonMark specification.",
```

Comment:
```suggestion
        description="Markdown text positioned below the `Graph.title`, compliant with the CommonMark specification.",
```
Just to avoid confusion with what component is really meant

Reply:
@lingyielia : I am curious if the word `subtitle` in the description would really help the model to populate this field?

@huong-li-nguyen If it make no difference, would it be possible to weave in the work subtitle into the description. It would be useful for human readers of the docstring too 🤔 ? Like "Markdown text (e.g. a subtitle) ..."

Reply:
Yes, will add it in!